src/components/MenuItem.tsx

```diff
@@ -246,7 +247,7 @@ function MenuItem(
         iconWidth,
         iconHeight,
         iconStyles,
-        fallbackIcon = Expensicons.FallbackAvatar,
+        fallbackIcon = undefined,
```

Comment:
@chrispader is there a better way to use a theme dependent image as a default prop? You can see how I did it here or in the `Avatar.tsx` file

Reply:
No, unfortunately not since we depend on the hook which is only executed within the render function...

We can remove the `= undefined` here though, it's unnecessary :)